Create Box developer glossary (#483)
box/developer.box.com: content/guides/getting-started/box-glossary.md

---
rank: 1
---

# Box developer glossary

This glossary includes terms and definitions used in Box Platform.

<!-- markdownlint-disable line-length -->

| Box Resource Type/Term | Also known as (AKA) | Description |
| --- | --- | --- |
| Admin | Superuser, administrator, supervisor | The main admin on an enterprise account. Admins can manage users and groups, view and edit all of their organization’s files and folders, log in to any user’s account within their organization, edit settings for their organization, and run or access reports. The Box Admin is the principal account administrator. |

> | Admin Console | | The dashboard area for administering a particular enterprise. Accessible by the admin, group admins and co-admins. |

Reviewer: ```suggestion
| Admin Console | | The dashboard area for administering a particular enterprise. Accessible by the admin, group admins, and co-admins. |
```

| Anonymous User | | A user that is not logged in. |

> | App User | | [App users][app user] are only accessible with the API - they do not have login credentials. They can be created by a Service Account and are only applicable to applications leveraging server to server authentication. App Users are tied to the application used to create them, and the user itself cannot be moved under another application. They can however collaborate on content outside of the application. |

Reviewer: ```suggestion
| App User | | [App Users][app user] are only accessible with the API - they do not have login credentials. They can be created by a Service Account and are only applicable to applications leveraging server-to-server authentication. App Users are tied to the application used to create them, and the user itself cannot be moved under another application. They can however collaborate on content outside of the application. |
```

| Bookmark / Web Link | Symbolic Link, Symlink, Soft Link, Reference, Relationship | |
| Box App Center | App Store | The first place for Box users to find out about applications that they can use in combination with Box. |

> | Box Command Line Interface | [Box CLI][CLI] | User-friendly command line tool allows both technical and non-technical users to leverage the Box API to perform routine or bulk actions. |

Reviewer: ```suggestion
| Box Command Line Interface | [Box CLI][CLI] | A user-friendly command line tool that allows both technical and non-technical users to leverage the Box API to perform routine or bulk actions. |
```

| [Box Embed][embed] | | An HTML-based framework that makes it possible to embed the Box Web App experience anywhere in the 3rd party applications. Box Embed provides the ability to upload, search, comment, share, tag, and edit files using Box Edit. |
| Box Platform | | An API-driven cloud content management and collaboration platform that provides developers with the tools and infrastructure to build custom applications, integrate workflows, and leverage powerful features for secure document storage, access control, versioning, metadata management, and real-time collaboration. |
| [Box Sample Code Catalog][code-catalog] | | Allows developers and admins to search through code repositories of tools, automation scripts, self paced workshops, and demo apps in multiple programming languages. The catalog contains more that eighty open source repositories and guides. |
| Box Shield | | A security solution offered by Box that helps protect sensitive data, prevent unauthorized access, and detect potential threats through automated classification and proactive monitoring. |

> | Box Skills | | Box Skills are AI-powered capabilities that enhance the functionality of Box by automatically extracting insights and metadata from content. |

Reviewer: I'm wondering if we should explain the difference between these Box Skills and Box (Custom) Skills.

| [Box Relay][relay] | | Box Relay allows you to create and configure workflow automations to automate and accelerate business processes centered around content. |
| [Box Verified Enterprise][bve] | BVE | Box Verified Enterprise is a certification program that ensures the security and compliance of an organization's content management system. |
| [Box UI Elements][ui-elements] | | Pre-built UI components that allow developers to add elements of the main Box web application into their own applications. They can be used to navigate through, upload, preview, and select content stored on Box and are available both as React components and framework-agnostic JavaScript libraries. |

> | Box Verified Enterprise | BVE | Box Verified Enterprise is a certification program that ensures the security and compliance of an organization's content management system. |

Reviewer: Already mentioned in line 27 :)

| [Collaborations][collaborations] | Permissions, Role-based Access Control (RBAC), Access Control List (ACL), Permission Mapping | Collaborations define access permissions for users and groups to files and folders, similar to access control lists. A collaboration object grants a user or group access to a file or folder with permissions defined by a specific role. |
| Collaborator | | Someone who shares access to a file or folder. |
| Collaborated folder | Shared folder | A shared folder owned by someone within your enterprise. |
| Collaboration Roles | Permission levels, access levels | The collaboration roles define the level of permissions a user has for a specific file or folder. The collaboration roles in Box are Owner, Co-owner, Editor, Viewer Uploader, Previewer Uploader, and Viewer. Collaboration levels in Box follow a "waterfall" design in which individuals have access only to the folder they are invited into and any sub-folders beneath it. You can also be invited to individual files. |
| Collection | | A collection of items, including files and folders. The contents of a collection can be explored in a similar way to which the contents of a folder is explored. |

> | [Content Manager][content-manager] | | A feature in the Admin Console that allows you to: search for and download  files and folders in your organization, browse by user to see the files and folders they can access, move files between folders, invite collaborators to folders, get shared links and modify access levels, and delete files and folders from any user’s Trash. |

Reviewer: ```suggestion
| [Content Manager][content-manager] | | A feature in the Admin Console that allows you to search for and download  files and folders in your organization, browse by user to see the files and folders they can access, move files between folders, invite collaborators to folders, get shared links and modify access levels, and delete files and folders from any user’s trash. |
```

> | [Co-admin][co-admin] | | A user other than the main admin, who has some subset of administrative privileges. Co-admins can perform the same duties as the organization’s Admin, but they cannot make changes to the Admin’s permissions or other Co-admins' permissions. The default access levels for Co-admins include only the ability to manage users and groups, but they can be modified on a per-user basis. |

Reviewer: ```suggestion
| [Co-admin][co-admin] | | A user other than the main admin, who has a subset of administrative privileges. Co-admins can perform the same duties as the organization’s Admin, but they cannot make changes to the Admin’s permissions or other Co-admins' permissions. The default access levels for Co-admins include only the ability to manage users and groups, but they can be modified on a per-user basis. |
```

| Custom App | | A tailored application that extends the functionality of Box by integrating with other tools and systems. It's used to streamline workflows and enhance collaboration. It can be created in the Developer Console and has several authentication methods available. |
| Developer | Programmer | A skilled professional who designs, builds, and maintains software applications or systems and has access to the Developer Console. In Box, the developer can have an Admin role. |
| Developer Console | | A portal that allows developers to create, debug, test, and monitor their applications by providing real-time insights into code execution and system performance. |
| Enterprise | Repository, Content Store, File Cabinet, Docbase, Vault | |
| [Ethical Walls][info-barriers] | Information Barriers | A mechanism that prevents exchanges or communication that could lead to conflicts of interest and therefore result in business activities ethically or legally questionable. |
| [Event][event] | Result of an action taken by the user. See [Using the Enterprise Event Stream][event-stream]. |
| External collaborator | External user | A collaborator who is not part of an enterprise. |

> | External collaborated folder | | Folder owned by someone outside your enterprise. |

Reviewer: ```suggestion
| External collaborated folder | | A folder owned by someone outside your enterprise. |
```

| Group | Team |  | Collaborative workspace where users can share files, collaborate on documents, and communicate with each other. |
| Item | Object, Box object, Content | Can represent a file, a folder or a web link. |
| Limited Access App | | Secure and restricted application created in the Developer Console that allows specific users to access and interact with designated content within the Box Platform. |
| Managed User | | A user account that is centrally controlled and administered by an organization's admin. |
| Metadata Template | Document Class, Document Type, Content Type, Indexes | A predefined structure that captures and organizes important information about a document or file. | 
| Metadata Attribute | Property, Field, Keyword, Index Value | A metadata attribute is a piece of information that provides descriptive details about data, such as its type, format, or source. |
| [Managed User][managed-users] | | A user account that is centrally controlled and administered by an organization's admin. |
| Personal folder | | A folder owned by an individual user. |

> | [Report][report] | | All Box business accounts are equipped with simple but powerful reporting functionality. From the Admin Console Reports tab you can run a variety of account-wide reports: usage logs, file/user statistics and security audits. |

Reviewer: ```suggestion
| [Report][report] | | A file containing a specific data set. You can use the Admin Console Reports tab to run a variety of account-wide reports: usage logs, file/user statistics, and security audits. |
```

| Sandbox | Testing environment | Managed, trackable, non-production, testing environment for developers. |
| [Service Account][service-account] | | A Service Account provides developers with a programmatic authentication mechanism for server-side integrations with Box. In other words, an application can authenticate to Box as the service, which is represented by a Service Account user. A Service Account can then be used to create other application specific users, called App Users. |

> | [Shared Link][shared-link] | | Hyperlink to content stored in Box you can share with your coworkers and friends – both inside and outside of the company. Sending someone a shared link to a file or folder is a way to work together with that person around shared content. Customizable permission levels, expiration dates, and optional password-protection make Box shared links a secure, simple way to share important content. |

Reviewer: ```suggestion
| [Shared Link][shared-link] | | A hyperlink to content stored in Box you can share with your coworkers and friends – both inside and outside of the company. Sending someone a shared link to a file or folder is a way to work together with that person around shared content. Customizable permission levels, expiration dates, and optional password-protection make Box shared links a secure, simple way to share important content. |
```

| User | Identity, person, authority | An individual who utilizes the collaborative document creation editor to create, edit, and share documents with others. |
| Unmanaged user | | A user account that is not centrally managed by an organization and has limited access and control over the content within Box. The unmanaged users may or may not also be external users. |
| Version control | | The management and tracking of different versions of a document, ensuring that everyone is working on the most up-to-date version. |
| Webhook | | A webhook is a way for an application to provide real-time data or notifications to another application by sending HTTP POST requests. |
| Workflow | | Workflows within Box are automated sequences of tasks that help streamline and track the progress of document-centric processes, ensuring efficient collaboration and timely completion. |

<!-- markdownlint-enable line-length -->

[app user]: g://getting-started/user-types/service-account

> [CLI]: https://www.box.com/platform

Reviewer: ```suggestion
[CLI]: https://github.com/box/boxcli
```
Maybe a more direct link?
